specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-03-01/amlfilesystem.json

```diff
@@ -0,0 +1,1679 @@
+          "importJobs_Delete": {
+            "$ref": "./examples/importJobs_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "tags": [
+          "importJobs"
+        ],
+        "description": "Schedules an import job for deletion.",
+        "operationId": "importJobs_Delete"
+      },
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AmlFilesystemNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ImportJobNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the import job object corresponding to amlFilesystemName.",
+            "schema": {
+              "$ref": "#/definitions/ImportJob"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "importJobs_Get": {
+            "$ref": "./examples/importJobs_Get.json"
+          }
+        },
+        "tags": [
+          "importJobs"
+        ],
+        "description": "Returns an import job.",
+        "operationId": "importJobs_Get"
+      },
+      "put": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AmlFilesystemNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ImportJobNameParameter"
+          },
+          {
+            "description": "Object containing the user-selectable properties of the import job. If read-only properties are included, they must match the existing values of those properties.",
```

Comment:
>description [](http://example.com/codeflow?start=13&length=11)

Would u like to call out any restrictions for this PUT operation? Are you going with the approach of adding a service defined resource instance to this collection which will be readOnly?

Reply:
Meaning would u reject the put for that specific named instance - maybe called initialImportJob?